CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## [0.4.0](https://github.com/auth0/Guardian.Android/tree/0.4.0) (2019-04-03)
+[Full Changelog](https://github.com/auth0/Guardian.Android/compare/0.3.0...0.4.0)
+
+**Added**
```

Comment:
We should probably put a note that this is specifically for moving from GCM to FCM here.

Maybe a high-level "This release is for compatibility with Firebase Cloud Messaging (FCM) which has replaced Google Cloud Messaging (GCM)."

Reply:
Added